Dockerfile

```diff
@@ -6,11 +6,10 @@ RUN apt update && \
     apt-get install -y --no-install-recommends socat
 
 # Create config and data dirs
-RUN mkdir config && \
-    mkdir antfarm-data
+RUN mkdir -p sia-antfarm/data
```

Comment:
What was there a reason to remove making the config dir? 

Also comment should be updated

Reply:
While I was fixing `antfarm-data` I have noticed the `config` dir is created by `COPY` command, so it is not needed here.